daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md

```diff
@@ -128,6 +134,42 @@ curl -X POST http://localhost:3500/v1.0/publish/myPulsar/myTopic?metadata.delive
       }'
 ```
 
+### OIDC Authentication
+
+Since `v3.0`, [Pulsar supports OIDC authentication](https://pulsar.apache.org/docs/3.0.x/security-openid-connect/).
+To enable OIDC authentication, you need to provide the following OAuth2 parameters to the component spec.
```

Comment:
Please add some paragraph breaks (empty line in between), or maybe even use bullet points. When rendered this all appears in one paragraph.